erpnext/stock/doctype/batch/batch.py

```diff
@@ -330,3 +330,9 @@ def get_active_batch(item_code):
 	active_batch = frappe.get_all("Batch", {"item": item_code, "display_on_website": 1})
 	active_batch = active_batch[0] if active_batch else {}
 	return active_batch
+
+@frappe.whitelist()
+def get_active_batch_coa(item_code):
+	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1}, fields=["name", "certificate_of_analysis"])
```

Comment:
Add the `fields` key to the `get_active_batch` function above instead, let's not make multiple endpoints for this. If you want, rename the function to `get_active_batch_details`